src/pages/Popup/index.css

```diff
@@ -0,0 +1,3 @@
+html {
```

Comment:
dont use css if it isnt needed add you can add attributes to the globalstyle which will include the css need
for example:
<GlobalStyle page={"popup"}> add the width of the popup in a separate styled.css

Reply:
idk how to do it